### PR TITLE
es-ES: Updated up to issue #1545

### PR DESCRIPTION
**Pull request by frenchiveruti**
Applying for issue(s):
- #1542 
- #1545

### CODE REVIEW

File: data/language/es-ES.txt [hunk 1]
```diff
@@ -1150,7 +1150,9 @@ STR_1770    :{SMALLFONT}{BLACK}Número de balanceos completos
 STR_1771    :{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{COMMA16}
 STR_1773    :Sólo se permite una sección de fotos por atracción.
 STR_1774    :Sólo un cable de elevación por atracción.
-STR_1777    :{WINDOW_COLOUR_2}Música de atracción
+STR_1775    :Off
```
Comment (AaronVanGeffen) [inline data/language/es-ES.txt]: These strings were deliberately removed from the language files, as they are not used. Please do not re-introduce them. :)

File: data/language/es-ES.txt [hunk 2]
```diff
@@ -3729,7 +3729,10 @@ STR_6258    :Transparente
 STR_6259    :Deshabilitado
 STR_6260    :Mostrar casillas bloqueadas
 STR_6261    :Mostrar caminos anchos
-
+STR_6262    :Volúmen Maestro
+STR_6263    :{SMALLFONT}{BLACK}Todo sonido on/off
```
Comment (Wirlie) [inline data/language/es-ES.txt]: "Activar o desactivar todo el sonido" is the appropriate translation.

**Comment by Wirlie** [comment]
Except for the string 6263, all other strings seems to be ok for me 👍

File: data/language/es-ES.txt (hunk 2 above)
Comment (frenchiveruti) [reply data/language/es-ES.txt]: You're right, I didn't realize it was a "message" rather than a GUI element.

**Comment by Gymnasiast** [comment]
I fixed the Git problem.

**Pull request merged**